modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +201,279 @@
+	can_suppress = FALSE
+	fire_sound = 'modular_skyrat/modules/sec_haul/sound/pistol_fire.ogg'
+	rack_sound = 'sound/weapons/gun/pistol/rack.ogg'
+	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
+	bolt_drop_sound = 'sound/weapons/gun/pistol/slide_drop.ogg'
+
+/obj/item/ammo_box/magazine/security_pistol
+	name = "influence security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with rubberized influence rounds."
+	icon = 'modular_skyrat/modules/goofsec/icons/gun_sprites.dmi'
+	icon_state = "security_pistol_influence"
+	base_icon_state = "security_pistol_influence"
+	ammo_type = /obj/item/ammo_casing/security_pistol/influence
+	caliber = CALIBER_SECURITYPISTOL
+	max_ammo = 20
+
+/obj/item/ammo_box/magazine/security_pistol/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state][ammo_count() ? "" : "-empty"]"
+
+/// Ammo for the Security Pistol.
+/obj/item/ammo_casing/security_pistol
+	name = "security pistol casing"
+	desc = "A casing for a security pistol magazine."
+	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
+	icon_state = "sl-casing"
+	caliber = CALIBER_SECURITYPISTOL
+	projectile_type = /obj/projectile/bullet/security_pistol
+
+/obj/projectile/bullet/security_pistol
+	name = "security pistol bullet"
+	damage = 0 // not supposed to exist
+
+// Influence rounds. Bouncy. Stamina only.
+/obj/item/ammo_box/magazine/security_pistol/influence
+	name = "influence security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with Influence rounds, which bounce around and influence people to comply with arrests."
+	icon_state = "security_pistol_influence"
+	base_icon_state = "security_pistol_influence"
+	ammo_type = /obj/item/ammo_casing/security_pistol/influence
+
+/obj/item/ammo_casing/security_pistol/influence
+	name = "influence security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/influence
+	harmful = FALSE
+
+/obj/projectile/bullet/security_pistol/influence
+	name = "influence security pistol bullet"
+	damage = 41
+	damage_type = STAMINA
+	ricochets_max = 6
+	ricochet_incidence_leeway = 0
+	ricochet_chance = 130
+	ricochet_decay_damage = 0.8
+	wound_bonus = CANT_WOUND
+
+// TRAC rounds. Lower raw damage. Puts a tracking chip in the target, embeds. Penetrates armor.
+/obj/item/ammo_box/magazine/security_pistol/trac
+	name = "trac security pistol magazine"
+	desc = "A magazine for the security pistol. Loaded with TRAC rounds, which embed a tracking chip in the target, while also being handy against armor."
+	icon_state = "security_pistol_trac"
+	base_icon_state = "security_pistol_trac"
+	ammo_type = /obj/item/ammo_casing/security_pistol/trac
+
+/obj/item/ammo_casing/security_pistol/trac
+	name = "trac security pistol casing"
+	projectile_type = /obj/projectile/bullet/security_pistol/trac
+
+/obj/projectile/bullet/security_pistol/trac
+	name = "trac security pistol bullet"
+	damage = 25
+	armour_penetration = 75
```

Comment:
Why does this get such high armour pen? It'd be better as a new round.

Reply:
.TRAC on the detective suffers from lack of use, so I figured I'd give it a boost and to give security an answer to armored enemies like Nuke Ops.